src/index.html

```diff
@@ -1,14 +1,19 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>Website</title>
   <base href="/">
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="Description" content="GP4Tech es una comunidad tecnológica, en la que mujeres involucradas con esta temática comparten sus conocimientos y experiencia. Lo que buscamos es incentivar y motivar a más mujeres a participar del área de tecnología, mediante charlas y actividades, organizadas por Ingenieras Jala">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <!-- <link rel="canonical" href="https://gp4techsite.firebaseapp.com"/> -->
```

Comment:
add a TODO for this explaining for what will be used in the future

Reply:
I'll delete this. I added it in the first place because Lighthouse was giving me the advice to add it. But I made the research and this is only needed when there are duplicated html files.